package.json

```diff
@@ -69,7 +69,7 @@
     "tslint": "^6.1.3",
     "typescript": "^4.2.4"
   },
-  "optionalPeerDependencies": {
+  "optionalDependencies": {
```

Comment:
This change means the package will attempt to be installed / built, but the install will not fail if it cannot be. Ideally we'd host packages somewhere for node-purple to avoid building it.

Reply:
This will be done in a future release.